tqecjs/src/workspace/index.js

```diff
@@ -0,0 +1,93 @@
+import { useApp } from '@pixi/react';
+import { Container } from 'pixi.js';
+import { makeGrid } from './grid';
+import Qubit from './QubitClass';
+import Tile from './TileClass';
+import { button } from './components/button';
+
+export default function TQECApp() {
+	// Initialize the app
+	let app = useApp();
+	// Remove all children from the stage to avoid rendering issues
+	app.stage.removeChildren();
+	const gridSize = 50;
+	// Let's create the workspace
+	const workspace = new Container();
+	workspace.name = 'workspace';
+	// Create the grid container
+	const grid = makeGrid(app, gridSize);
+
+	workspace.addChild(grid);
+	// Add the qubits to the workspace
+	for (let x = 0; x <= app.renderer.width; x += gridSize) {
+		for (let y = 0; y <= app.renderer.height; y += gridSize) {
+			// Skip every other qubit
+			if (x % (gridSize * 2) === 0 && y % (gridSize * 2) === 0) continue;
+			if (x % (gridSize * 2) === gridSize && y % (gridSize * 2) === gridSize)
+				continue;
+			// Create a qubit
+			const qubit = new Qubit(x, y, 5, gridSize);
+			// Name the qubit according to its position
+			qubit.name = `${x}_${y}`;
```

Comment:
I think this was discussed earlier. We could have a name() function (like toString) that can return an absolute grid position, or relative to its neighbors in the plaquette. Might be worth discussing in our next meeting.

Reply:
it would have to be a different name than name() since name is already a built in graphic property. ill think of something that works with your suggestion :)